vignettes/deployment.Rmd

```diff
@@ -165,7 +156,7 @@ if (Sys.getenv("id_rsa") != "") {
     add_step(step_setup_ssh())
 }
 
-if (Sys.getenv("BUILD_PKGDOWN") != "") {
+if (isTRUE("BUILD_PKGDOWN")) {
```

Comment:
``` r
isTRUE("true")
#> [1] FALSE
```

<sup>Created on 2018-11-27 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1.9000)</sup>